src/shaders/SkPictureShader.cpp

```diff
@@ -298,6 +298,16 @@ sk_sp<SkShader> SkPictureShader::rasterShader(const SkMatrix& viewMatrix,
         }
 
         SkResourceCache::Add(new ImageFromPictureRec(key, image));
+
+        // Track SkPicture cache size (so we get a general idea of memory impact)
+        SkRecordReplayPrint(
```

Comment:
Note: We don't have easy access to this information when we leak, so we print it here for now.